app/printing/printSimplified/get.controller.js

```diff
@@ -21,9 +23,26 @@ const printSimplifiedSentencePlan = async ({ path, params: { id, planId }, token
       return res.render('app/error', { error })
     }
 
+    // only get intervention data if there is any objective with an action with an intervention
+    const hasInterventions = objectives.some(objective => {
+      return objective.actions.some(({ intervention }) => intervention)
+    })
+
+    let interventionList = []
+    if (hasInterventions) {
+      interventionList = await getInterventions(tokens)
+    }
+
     objectives.forEach(objective => {
       const currentObjective = objective
       currentObjective.type = getObjectiveType(currentObjective)
+      objective.actions.map(action => {
+        const tempAction = action
```

Comment:
This mutates the original `objective` elements. Nothing wrong per se, but might cause issues if the raw data is required again. I'd be tempted to use a `forEach` as that implies mutation, where as `map` sort of implies immutability.